pkg/pxc/app/statefulset/node.go

```diff
@@ -308,3 +312,15 @@ func (c *Node) UpdateStrategy(cr *api.PerconaXtraDBCluster) appsv1.StatefulSetUp
 		}
 	}
 }
+
+func (c *Node) SetSfsSpecTemplateAnnotations(annotations map[string]string) {
+	if len(annotations) == 0 {
+		return
+	}
+	if c.sfs.Spec.Template.Annotations == nil {
+		c.sfs.Spec.Template.Annotations = make(map[string]string)
+	}
+	for k, v := range annotations {
+		c.sfs.Spec.Template.Annotations[k] = v
+	}
+}
```

Comment:
Look comment for `Proxy.SetSfsSpecTemplateAnnotations`

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/495/commits/8e13ea090654a503cdbde00eb0ee4a83b791d608